package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Webtrader",
-  "version": "2.2.8",
+  "version": "2.2.9",
```

Comment:
same here 🙏 , we dont need to bump new version here

Reply:
Sure, will change Webtrader version to 2.2.8